File: app/src/main/java/com/elseboot3909/GCRClient/PreLoader.java
package com.elseboot3909.GCRClient;

import android.app.Application;

import com.google.android.material.color.DynamicColors;

public class PreLoader extends Application {

    @Override
    public void onCreate() {
        super.onCreate();
        DynamicColors.applyToActivitiesIfAvailable(this);
    }
}

File: app/src/main/java/com/elseboot3909/GCRClient/PreLoader.kt
@file:Suppress("unused")

package com.elseboot3909.GCRClient

import android.app.Application

import com.google.android.material.color.DynamicColors

class PreLoader: Application() {

    override fun onCreate() {
        super.onCreate()
        DynamicColors.applyToActivitiesIfAvailable(this)
    }
}